resources/shaders/shadow_ray.hlsl

```diff
@@ -60,7 +60,6 @@ float GetShadowFactor(float3 wpos, float3 light_dir, float t_max, uint depth, in
 	float shadow_factor = 0.0f;
 
 #ifdef SOFT_SHADOWS
-	[unroll(MAX_SHADOW_SAMPLES)]
```

Comment:
Conflicts with this and your previous branch; why is this unroll removed, does it not increase performance?

Reply:
It crashes fallback.